TradeItIosTicketSDK2/TradeItYahooViewController.swift

```diff
@@ -3,6 +3,7 @@ import UIKit
 class TradeItYahooViewController: CloseableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
+        guard let yahooNavigationController = self.navigationController as? TradeItYahooNavigationController else { return }
```

Comment:
Not 100% sure what to do here - default to a number? It should never happen but it might.